### ISSUE TITLE
Error message is not displayed after clicking ‘Send’ button on without entering value in 'Send a message' edit field.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4623]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who depend on keyboard will face difficulty as Error message is not displayed after clicking ‘Send’ button on without entering value in 'Send a message' edit field.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Send box

### What is the public URL for the website?

https://web.powerva.microsoft.com/environments/31c9fd16-d6ad-4925-8549-2d578a71b990/bots/a619e4ca-7ccb-4da9-a2c6-48fcba807887/canvas

### How to reproduce the issue?

1. Open the URL https://web.powerva.microsoft.com/environments/31c9fd16-d6ad-4925-8549-2d578a71b990/bots/a619e4ca-7ccb-4da9-a2c6-48fcba807887/canvas in Edge Browser.
2. Microsoft Power Platform home page gets displayed.
3. Navigate to Type your message edit field in ‘Device bot virtual agent’ chat dialog.
4. Do not provide any data in Type your message edit field and navigate to Send button and invoke it.
5. Verify whether error message is displayed or not after invoking ‘Send’ button.

### What do you expect?

Error message should be displayed after clicking ‘Send’ button on without entering value in 'Send a message' edit field.
**Ex:** After clicking 'Send' button on without entering value in type your message edit field, Error message should display as "Please type as message to continue"
**Ensure that** one the error message is displayed screen reader is narrating the message displayed.

### What actually happened?

Error message is not displayed after clicking ‘Send’ button on without entering value in 'Send a message' edit field.
**Current Observation:** After clicking send button on without entering value in type your message edit field, Error message is not displayed.

### MAS reference

https://aka.ms/MAS3.3.1

### Additional context

Copied from https://msazure.visualstudio.com/CCI/_workitems/edit/16393475.
